awd/cli.py

```diff
@@ -35,24 +37,24 @@ def create_list_of_dspace_item_files(
 
 
 def doi_to_be_added(doi: str, doi_items: list[dict[str, Any]]) -> bool:
-    "Validate that a DOI is not a part of the database table and needs to  be added."
+    """Validate that a DOI is not a part of the database table and needs to  be added."""
     validation_status = False
     if not any(doi_item["doi"] == doi for doi_item in doi_items):
         validation_status = True
-        logger.debug(f"{doi} added to database.")
+        logger.debug("%s added to database", doi)
     return validation_status
 
 
 def doi_to_be_retried(doi: str, doi_items: list[dict[str, Any]]) -> bool:
-    "Validate that a DOI should be retried based on its status in the database table."
+    """Validate that a DOI should be retried based on its status in the database table."""
     validation_status = False
-    for doi_item in [
+    for _doi_item in [
         d
         for d in doi_items
         if d["doi"] == doi and d["status"] == str(Status.UNPROCESSED.value)
     ]:
         validation_status = True
-        logger.debug(f"{doi} will be retried.")
+        logger.debug("%s will be retried", doi)
     return validation_status
```

Comment:
I might propose using the same logic and flow as the similar function `doi_to_be_added()` here.  I think the linting suggestion that `doi_item` is not used is a good indicator -- instead of just adding an underscore `_` -- that this could be reworked to avoid that variable entirely.

The pattern of `any(...)`, like the other similar function, could be applied here.

Reply:
Another great call, thanks!